pom.xml

```diff
@@ -141,7 +141,7 @@
                 <plugin>
                     <groupId>org.eclipse.dash</groupId>
                     <artifactId>license-tool-plugin</artifactId>
-                    <version>1.0.3-SNAPSHOT</version>
+                    <version>1.0.2</version>
```

Comment:
the version needs to be -SNAPSHOT.
It is removed during the release workflow

Reply:
@shaishap we need to change to a release version of org.eclipse.dash cuz if we reference a snapshot when we try and run the release maven plugin, the workflow fails stating "release versions can only pull in other release versions of dependency projects" (or something like that)